### ISSUE TITLE
Differentiate third-party apps by openness

### ISSUE THREAD

**Issue by UnlimitedCookies** [mastodon/joinmastodon#369]
Currently the app selector only indicates whether an app is free monetarily or paid.
In my opinion, another filter category should be added to filter for apps that are free and open source and adhere to standard open source practices.

**Comment by varun-s22** [comment]
Hey @connyduck can i take on this? I'm new to this, and I would love to work on this :)

**Comment by connyduck** [comment]
@varun-s22 Why are you asking me? I am just a random contributor and have nothing to say here.

**Comment by varun-s22** [comment]
> @varun-s22 Why are you asking me? I am just a random contributor and have nothing to say here.

oh sorry :(
i'm relatively new here, and i thought you might be reviewer or collaborator

sorry for the inconvenience caused

**Comment by andypiper** [comment]
I like the idea in general, but I have a small concern that we may be adding too many options and features to what was intended to be a (relatively) simple list.

There's also a proposal (#326) to do more categorisation by desktop OS. This *could* be something that could be rolled together into the same piece of work to refresh how the apps list is managed.

We have limited capacity for doing more significant changes like this, so we are definitely open to reviewing community contributions and ideas - it is not something that we will be able to work on right away. Thanks for your support and understanding!

**Comment by andypiper** [comment]
You're free to submit a PR here, but as mentioned I'm not sure that it is anything that we will have time to focus on immediately. If you want to see if there is a collaboration opportunity with the other issues, that's also an option - I do not have a strong preference.

**Comment by varun-s22** [comment]
> You're free to submit a PR here, but as mentioned I'm not sure that it is anything that we will have time to focus on immediately. If you want to see if there is a collaboration opportunity with the other issues, that's also an option - I do not have a strong preference.

yeah sure.. i mean i kind of linked them as good first issues to solve on..
I'll look into others as well
do you guys think there's a particular issue to work on, like some kind of preference? or should i take any thing?

sorry to bother you tho :)

**Comment by andypiper** [comment]
Feel free to work on anything you feel equipped to look at! I will try to review as needed, but have a number of things to work on elsewhere. Thank you for looking for ways to get involved.

**Comment by ChaosExAnima** [comment]
closing this as resolved by #855.
